src/db/db_user.erl

```diff
@@ -0,0 +1,32 @@
+-module(db_user).
+
+-export([create/1,
+         authenticate/2,
+         get_from_countries/1]).
+
+create(#{<<"email">> := Email,
```

Comment:
Using atoms makes sense, I guess that should be part of the validation/sanitization part of the input (which I didn't get to yet), turning into atoms only the fields we expect (so we're not creating a bunch of atoms if garbage input comes in).